src/common/commands.ts

```diff
@@ -160,18 +160,54 @@ export async function executeServerDefinitionCommand(lsClient: LanguageClient |
     }
 }
 
-export async function executeGetProjectDataCommand(lsClient: LanguageClient | undefined) {
+export async function executeGetProjectDataCommand(
+    lsClient: LanguageClient | undefined,
+    pipelineName: string | undefined = undefined,
+) {
     if (!lsClient || lsClient.state !== State.Running) {
         await vscode.window.showErrorMessage('There is no language server running.');
         return;
     }
     if (!lsClient.initializeResult) {
-        await vscode.window.showErrorMessage('The Language Server fail to initialise.');
+        await vscode.window.showErrorMessage('The Language Server failed to initialize.');
```

Comment:
NIT: I think it's better to follow UK spelling "initialise" as thats what we have the doc too